Dockerfile

```diff
@@ -8,6 +8,8 @@ RUN apt-get update && apt install -y --no-install-recommends build-essential
 RUN pip install poetry
 COPY pyproject.toml poetry.lock /app/
 
-RUN poetry export --without-hashes -f requirements.txt  | pip install -r /dev/stdin
+#primarily used for installing pytest
+ARG INSTALL_DEV
```

Comment:
@snejus so I'm now using  `--build-arg` flag to add `--dev`. Should be okay?

Reply:
![](https://media.giphy.com/media/xTiQyBOIQe5cgiyUPS/giphy.gif)

Reply:
What happens if I add `--build-arg '; sudo rm -rf;' ?

Reply:
the same could be done with any Makefile commands. Are we over engineering this?